app/components/footer_component.rb

```diff
@@ -1,20 +1,11 @@
 class FooterComponent < ViewComponent::Base
-  def initialize(talk_to_us: true, feedback: false)
+  def initialize(talk_to_us: true)
     super
 
     @talk_to_us = talk_to_us
-    @feedback = feedback
   end
 
   def talk_to_us?
     @talk_to_us
   end
-
-  def feedback?
-    @feedback
-  end
-
-  def mailing_list?
```

Comment:
@martyn-w As per my other comment - why are we removing this as well? Is it not used?

Reply:
it is not used and can be safely removed.